src/compiler/rust.rs

```diff
@@ -1445,6 +1502,11 @@ where
                         } else {
                             None
                         };
+                        if let Some(gcno) = gcno {
+                            let p = output_dir.join(&gcno);
+                            outputs.insert(gcno.to_string_lossy().into_owned(), p.clone());
```

Comment:
Clippy is complaining with
`error: redundant clone`
could you please fix that?

Reply:
done